script/model/training_util/data_transforming_util.py

```diff
@@ -105,6 +105,7 @@ def _tuple_num_log_predict_transform(x, y):
     OpUnit.OP_DECIMAL_DIVIDE: _tuple_num_linear_transformer,
     OpUnit.OP_DECIMAL_COMPARE: _tuple_num_linear_transformer,
     OpUnit.OUTPUT: _tuple_num_linear_transformer,
+    OpUnit.CREATE_INDEX: _tuple_num_linear_transformer,
```

Comment:
I think this should be `linear_log_transformer`?

Reply:
Yes. This should be `linear_log_transformer`.